[
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/352241790",
    "html_url": "https://github.com/jupyter/notebook/issues/3143#issuecomment-352241790",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/3143",
    "id": 352241790,
    "node_id": "MDEyOklzc3VlQ29tbWVudDM1MjI0MTc5MA==",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-12-17T09:10:57Z",
    "updated_at": "2017-12-17T09:10:57Z",
    "author_association": "MEMBER",
    "body": "It's looking for the [traitlets.utils.bunch](https://github.com/ipython/traitlets/blob/master/traitlets/utils/bunch.py) module. Your installation appears to be corrupted somehow. Try uninstalling and reinstalling the `traitlets` package. If that doesn't work, it may be necessary to delete Anaconda and install it again.",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/352279341",
    "html_url": "https://github.com/jupyter/notebook/issues/3143#issuecomment-352279341",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/3143",
    "id": 352279341,
    "node_id": "MDEyOklzc3VlQ29tbWVudDM1MjI3OTM0MQ==",
    "user": {
      "login": "govinda-kamath",
      "id": 7320464,
      "node_id": "MDQ6VXNlcjczMjA0NjQ=",
      "avatar_url": "https://avatars.githubusercontent.com/u/7320464?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/govinda-kamath",
      "html_url": "https://github.com/govinda-kamath",
      "followers_url": "https://api.github.com/users/govinda-kamath/followers",
      "following_url": "https://api.github.com/users/govinda-kamath/following{/other_user}",
      "gists_url": "https://api.github.com/users/govinda-kamath/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/govinda-kamath/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/govinda-kamath/subscriptions",
      "organizations_url": "https://api.github.com/users/govinda-kamath/orgs",
      "repos_url": "https://api.github.com/users/govinda-kamath/repos",
      "events_url": "https://api.github.com/users/govinda-kamath/events{/privacy}",
      "received_events_url": "https://api.github.com/users/govinda-kamath/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-12-17T19:31:30Z",
    "updated_at": "2017-12-17T19:32:15Z",
    "author_association": "NONE",
    "body": "That (uninstalling and reinstalling `traitlets`) did the trick. Thanks a lot!",
    "performed_via_github_app": null
  }
]
